display_list/display_list.cc

```diff
@@ -75,7 +75,6 @@ void DisplayList::ComputeRTree() {
 void DisplayList::Dispatch(Dispatcher& dispatcher,
                            uint8_t* ptr,
                            uint8_t* end) const {
-  TRACE_EVENT0("flutter", "DisplayList::Dispatch");
```

Comment:
Let's delete this and the one on `DisplayListDispatcher::EndRecordingAsPicture` in their own PR.